src/main/java/teammates/logic/core/InstructorsLogic.java

```diff
@@ -143,6 +143,15 @@ public boolean isEmailOfInstructorOfCourse(String instructorEmail, String course
         return instructorsDb.getInstructorForEmail(courseId, instructorEmail) != null;
     }
 
+    /**
+     * Returns whether at least one instructor is displayed to the students.
+     */
+    public boolean isNoInstructorDisplayedToStudentsAfterEditing(String courseId, boolean isEditedInstructorDisplayed) {
```

Comment:
1. Do you need this to be `public`?
1. If this method is only called once, why not just combine it with `verifyAtLeastOneInstructorIsDisplayed`?

Reply:
Yes, I can combine this with ` verifyAtLeastOneInstructorIsDisplayed`